### PR TITLE
ResourceMarkdown integration

### PR DESCRIPTION
**Pull request by myluki2000**
## Description of changes
Please provide a short description of the change or if the change is sufficiently described in the GitHub issue, provide a link to the issue.

  
## How has this been tested?
Please describe the test strategy you followed.

- [ ] automated unit test
- [ ] automated integration test
- [ ] automated acceptance test
- [x] manual, exploratory test

Tested querying the new field ResourceMarkdown->referencedMediaRecords
    
## Checklist before requesting a review
- [x] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code fulfills all acceptance criteria
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [x] I have made corresponding changes to the documentation
- [ ] I have added explanation of architectural decision and rationales to [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I have updated the changes in the ticket description

## Checklist for reviewer
- [ ] The code works and does not throw errors
- [ ] The code is easy to understand and there are no confusing parts
- [ ] The code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] The code change accomplishes what it is supposed to do
- [ ] I cannot think of any use case in which the code does not behave as intended
- [ ] The added and existing tests reasonably cover the code change
- [ ] I cannot think of any test cases, input or edge cases that should be tested in addition
- [ ] Description of the change is included in the documentation

### CODE REVIEW

File: additionalTypeDefs/ResourceMarkdown.graphqls [hunk 1]
```diff
@@ -0,0 +1,13 @@
+extend type ResourceMarkdown {
+    """
+    Resolved MediaRecords referenced in the ResourceMarkdown text in order.
+    A MediaRecord might be NULL if the referenced MediaRecord no longer exists. 
+    """
+    referencedMediaRecords: [MediaRecord]! @resolveTo(
+        sourceName: "MediaService",
+        sourceTypeName: "Query",
+        sourceFieldName: "mediaRecordsByIds",
```
Comment (PaulBredl) [inline additionalTypeDefs/ResourceMarkdown.graphqls:9]: Does this still exist? Didn't you rename it to "findMediaRecordsByIds"?

Comment (myluki2000) [reply additionalTypeDefs/ResourceMarkdown.graphqls:9]: Both queries still exist but you are correct in that this resolver should use the find... query because media records can be non-existant

**Review by PaulBredl — APPROVED** [review]
